dist/spec.yaml

```diff
@@ -14294,6 +15014,56 @@ definitions:
         type: array
     x-okta-tags:
       - Application
+  Subscription:
+    properties:
+      _links:
+        additionalProperties:
+          type: object
+        readOnly: true
+        type: object
+      channels:
+        items:
+          type: string
+        type: array
+      notificationType:
+        $ref: '#/definitions/NotificationType'
+      status:
+        $ref: '#/definitions/SubscriptionStatus'
+    type: object
+    x-okta-operations:
+      - alias: getRoleSubscriptionByNotificationType
+        operationId: getRoleSubscriptionByNotificationType
+      - alias: getRoleSubscriptions
+        operationId: getRoleSubscriptions
+      - alias: getCustomRoleSubscriptions
+        operationId: getCustomRoleSubscriptions
+      - alias: getCustomRoleSubscriptionByNotificationType
+        operationId: getCustomRoleSubscriptionByNotificationType
+      - in: getUserSubscriptionByNotificationType
```

Comment:
Should the `alias` be here instead of `in`?

Reply:
fixed, thank you @VitaliiTytarenko-okta